db/migrate/20160114200145_add_mirror_on_sync_to_repositories.rb

```diff
@@ -0,0 +1,6 @@
+class AddMirrorOnSyncToRepositories < ActiveRecord::Migration
+  def change
+    add_column :katello_repositories, :mirror_on_sync, :boolean, :default => true, :null => false
+    update "UPDATE katello_repositories SET mirror_on_sync = 'f'"
```

Comment:
Ahh, although, if you are redefining the model in the migration, how would
a save trigger Pulp? That way the ORM is used?
On Jan 15, 2016 5:57 PM, "Partha Aji" <notifications@github.com> wrote:

> In db/migrate/20160114200145_add_mirror_on_sync_to_repositories.rb
> <https://github.com/Katello/katello/pull/5709#discussion_r49915432>:
>
> > @@ -0,0 +1,6 @@
> > +class AddMirrorOnSyncToRepositories < ActiveRecord::Migration
> > +  def change
> > +    add_column :katello_repositories, :mirror_on_sync, :boolean, :default => true, :null => false
> > +    update "UPDATE katello_repositories SET mirror_on_sync = 'f'"
>
> Yeah you dont want repository.save to cause save happening in pulp. This
> is the safe way :). Infact there is a precedent for this look at
> https://github.com/Katello/katello/blob/master/db/migrate/20151203230940_add_date_type_to_content_view_erratum_filter_rules.rb#L4
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Katello/katello/pull/5709/files#r49915432>.
>